### PR TITLE
docs: re-organize images for blog post

### PR DESCRIPTION
**Pull request by tianzhou**
* Put blog images under /static/blog and each post has its own folder
* Unify images to use WebP and for most images, use the consistent 960px width
* Remove obsolete images

### CODE REVIEW

File: content/blog/how-bytebase-uses-render.md [hunk 1]
```diff
@@ -2,18 +2,16 @@
 title: How Bytebase uses Render
 author: Zilong
 published_at: 2022/02/21 08:08:57
-feature_image: /static/blog-changelog-assets/2022/02/mohamed-nohassi-odxB5oIG_iA-unsplash.jpg
-tags: Education
```
Comment (boojack) [inline content/blog/how-bytebase-uses-render.md:6]: The `tags` is removed, maybe by mistake?

**Comment by tianzhou** [comment]
PTAL

File: content/blog/introducing-bb-Bytebase-CLI-to-manage-database-operations.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Introducing bb - a Bytebase CLI tool to manage database operations"
 author: Candy
 published_at: 2022/05/27 09:57:57
-feature_image: /static/blog-changelog-assets/2022/05/CLI_bb.png
+feature_image: /static/blog/introducing-bb-bytebase-cli-to-manage-database-operations/bb.webp
```
Comment (boojack) [inline content/blog/introducing-bb-Bytebase-CLI-to-manage-database-operations.md:5]: Not found image

image

Looks like the related static folder name is `/introducing-bb-Bytebase-CLI-to-manage-database-operations/`.

Comment (tianzhou) [reply content/blog/introducing-bb-Bytebase-CLI-to-manage-database-operations.md:5]: Seems to be there

https://github.com/bytebase/bytebase.com/pull/289/files#diff-48b17e21ac7f8c96014029aa7428eeb240c27dcf016dd20dd130f326ad35735f

CleanShot 2022-06-27 at 09 36 56@2x

**Review by tianzhou — COMMENTED** [review]
PTAL

**Review by tianzhou — COMMENTED** [review]
Updated the case. PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**